packages/auth/src/access/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { isDeepStrictEqual } from 'util'
```

Comment:
how did I not realize this was in the node standard library? Cool.

Reply:
I just discovered it when completing this PR. I knew there surely was a library that could handle this but I wasn't expecting a node-native solution either!